roles/fix_python_encodings/tasks/main.yaml

```diff
@@ -0,0 +1,69 @@
+---
+- name: Check if cp437 is available
+  ansible.builtin.raw: |-
+    python3 -c 'from encodings import cp437; print(cp437)'
+  register: _import_cp437
+  changed_when: false
+  ignore_errors: true
+
+- name: Fix missing cp437
+  when: _import_cp437 is not success
+  block:
+    - name: Install python3-libs
+      ansible.builtin.raw: |-
```

Comment:
it really can not be command or shell? Almost sure that dnf module can handle that too